bom/csv_headers.py

```diff
@@ -234,8 +238,11 @@ class PartsListCSVHeadersSemiIntelligent(PartsListCSVHeaders):
         CSVHeader('part_class', name_options=['class', 'part_category']),
         CSVHeader('part_number', name_options=['part number', 'part no', ]),
         CSVHeader('revision', name_options=['rev', 'part_revision', ]),
-    ] + PartsListCSVHeaders.part_attributes
-
+    ] + PartsListCSVHeaders.part_attributes \
+      + SellerPartCSVHeaders.all_headers_defns + [
+        CSVHeader('lead_time_days', name_options=['part_lead_time_days', ]),
+        CSVHeader('minimum_pack_quantity', name_options=['part_minimum_pack_quantity', 'mpq' ]),
+    ]
```

Comment:
This is on me, I need to add some pre-commit hooks for pep-8 linting or something but there should be two returns between classes.

Reply:
I'll take care of this for my next contributions. I don't know why, but `flake8` was not installed on my system anymore.
I'm using `flake8` as a linter with ale vim. So such kind of problem should be highlighted.

I'll check where flake8 gone